data/recipes.yaml

```diff
@@ -48,6 +48,29 @@
   out:
   - [1, drill]
 
+- in:
```

Comment:
Oh, I think I get it --- the idea is that even though you can never actually have a `boulder`, `mountain`, or `copper vein` in your inventory, once you `scan` them you can see the recipe which hints that you should use a `drill`?  I like that a lot.

Reply:
Yes, I think I will add a note to the YAML explaining this assumption :slightly_smiling_face: The recipes do look nice :heart_eyes: 
![Screenshot from 2021-10-05 23-18-32](https://user-images.githubusercontent.com/44544735/136104313-9c347251-017b-4230-b7be-835b92c477aa.png)

Reply:
Actually, we are missing a map from requirements to recipes, so `drill` and `furnace` do not have recipes with them shown. I am thinking about leaving that improvement for later Issue/PR :sweat_smile: